cookbook/deployment/cluster/productionize_cluster.py

```diff
@@ -5,195 +5,127 @@
+be done independently from the sandbox deployment:
+
+* The kubernetes cluster needs to run securely and robustly
+* The sandbox's object store must be replaced by a production grade storage system
+* The sandbox's PostgreSQL database must be replaced by a production grade deployment of postgres
+* A production grade task queueing system must be provisioned and configured
+* A production grade notification system must be provisioned and configured
+* All the above must be done in a secure fashion
+* (Optionally) An official dns domain must be created
+* (Optionally) A production grade email sending system must be provisioned and configured
+
+A Flyte user may provision and orchestrate this setup by themselves, but the Flyte team has partnered with the
+`Opta <https://github.com/run-x/opta>`_ team to create a streamlined production deployment strategy for AWS with
+ready-to-use templates provided in the `Flyte repo <https://github.com/flyteorg/flyte/tree/master/opta>`_. The following
+documentation specifies how to use and further configure them.
+
+Deploying Opta Environment and Service for Flyte
+************************************************
+**The Environment**
+To begin using Opta, please first `download the latest version <https://docs.opta.dev/installation/>`_ and all the listed
+prerequisites and make sure that you have
+`admin/fullwrite AWS credentials setup on your terminal <https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html>`_.
+With that prepared, go to the Opta subdirectory in the Flyte repo, and open up env.yaml in your editor. Please find and
+replace the following values with your desired ones:
+
+* <account_id>: your AWS account ID
+* <region>: your AWS region
+* <domain>: your desired domain for your Flyte deployment (should be a domain which you own or a subdomain thereof - this environment will promptyly take ownership of the domain/subdomain so make sure it will only be used for this purpose)
+* <env_name>: a name for the new isolated cloud environment which is going to be created (e.g. flyte-prod)
+* <your_company>: your company or organization's name
+
+Once complete please run `opta apply -c env.yaml` and follow the prompts.
+
+**DNS Delegation**
+Once Opta's apply for the environment is completed, you will need to complete dns delegation to fully setup public
+traffic access. You may find instructions on `how to do so here <https://docs.opta.dev/miscellaneous/ingress/>`_.
+
+**The Flyte Deployment**
+Once dns deployment delegation is complete, you may deploy the Flyte service and affiliated resources. Go to the Opta
+subdirectory in the Flyte repo, and open up flyte.yaml in your editor. Please find and replace the following values with
+your desired ones:
+
+* <account_id>: your AWS account ID
+* <region>: your AWS region
+
+Once complete please run ``opta apply -c flyte.yaml`` and follow the prompts.
```

Comment:
can you do the double `` everywhere you are running commands? t hey render better...

Reply:
done!